src/calculateRentalCost.js

```diff
@@ -28,6 +28,18 @@
  */
 
 function calculateRentalCost(days) {
+  let totalCost = days * 40;
+
+  if (days > 6) {
+    totalCost -= 50;
+  }
+
+  if (days > 2 && days < 7) {
+    totalCost -= 20;
+  }
```

Comment:
math works, but 20 and 50 are currently what's called `magic numbers`
better be explained = introduced as variables with meaningful names.